lib/pure/oids.nim

```diff
@@ -67,12 +67,6 @@ template toStringImpl[T: string | cstring](result: var T, oid: Oid) =
   when T is cstring:
     result[N] = '\0'
 
-proc oidToString*(oid: Oid, str: cstring) {.deprecated: "unsafe; use `$`".} =
```

Comment:
The proc hasn't been deprecated in the major release(which is deprecated on devel branch), so better not to remove it. It is a breaking change.

Reply:
yes, same comment as https://github.com/nim-lang/Nim/pull/18022#discussion_r633028291, you have to check `git log` to see when was something deprecated before trying to remove it.

If you want to do cleanup PR's that are not problematic, there are plenty of useful things to do, we can discuss that